src/analysis/retail/warlock/affliction/modules/spells/SoulConduit.tsx

```diff
@@ -4,14 +4,13 @@ import TALENTS from 'common/TALENTS/warlock';
 import Analyzer, { SELECTED_PLAYER, Options } from 'parser/core/Analyzer';
 import Events, { DamageEvent } from 'parser/core/Events';
 import { findMax, binomialPMF } from 'parser/shared/modules/helpers/Probability';
-import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
 import Statistic from 'parser/ui/Statistic';
 import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import TalentSpellText from 'parser/ui/TalentSpellText';
 
 import SoulShardTracker from '../resources/SoulShardTracker';
 
-const TICKS_PER_UA = 4;
-const SC_PROC_CHANCE = 0.15;
+const SC_PROC_CHANCE_BASE = 0.05;
 
 class SoulConduit extends Analyzer {
```

Comment:
now that this in the class tree it seems like these should be merged and I could just set what "you were refunded enough shards for x more casts of y spender" message shows based on the combatant spec, but then it'd just be one file with a lot going on. not sure which way of organizing it is preferable.

Reply:
if its easy, generalizing is good. however, i'd rather have 3 simple modules (which can import shared constants) than 1 complicated module that tries to do everything.